Fiscal sponsor ci (#1095)
USRSE/usrse.github.io: pages/about/finances.md

set_last_modified: true
---

> ### Community Initiatives

Reviewer: Do we need to put "member" or anything like that?

Author: How is that?

<img src="{{ site.baseurl }}/assets/img/ci-secondary-600.png" alt="CI Logo" style="padding:60px; float:right; width:400px;">

> US-RSE is a fiscally sponsored project of [Community Initiatives](http://communityin.org/) (CI), a registered 501(c)(3) non-profit organisation based in California, USA. The 501(c)(3) status of CI and all of its benefits extend to all CI projects including US-RSE. 

Reviewer 2: ```suggestion
US-RSE is a fiscally sponsored project of [Community Initiatives](http://communityin.org/) (CI), a 501(c)(3) non-profit organisation based in California, USA. The 501(c)(3) status of CI and all of its benefits extend to all CI projects including US-RSE. 
```

> CI provides services such as accounting, book-keeping, and HR and legal fiscal services for tax purposes to US-RSE. 

Reviewer 2: ```suggestion
CI provides services such as accounting, bookkeeping, and HR and legal fiscal services for tax purposes to US-RSE. 
```

> For information on current fundraising initiatives and how to donate to US-RSE see the [support US-RSE page]({{ site.baseurl }}/support/).

Reviewer: ```suggestion
For information on current fundraising initiatives and how to donate to US-RSE, see the [support US-RSE page]({{ site.baseurl }}/support/).
```
